jcommune-performance-tests/src/test/scala/org/jtalks/jcommune/performance/tests/Login.scala

```diff
@@ -0,0 +1,44 @@
+/**
+  * Copyright (C) 2011  JTalks.org Team
+  * This library is free software; you can redistribute it and/or
+  * modify it under the terms of the GNU Lesser General Public
+  * License as published by the Free Software Foundation; either
+  * version 2.1 of the License, or (at your option) any later version.
+  * This library is distributed in the hope that it will be useful,
+  * but WITHOUT ANY WARRANTY; without even the implied warranty of
+  * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+  * Lesser General Public License for more details.
+  * You should have received a copy of the GNU Lesser General Public
+  * License along with this library; if not, write to the Free Software
+  * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+  */
+package org.jtalks.jcommune.performance.tests
+
+import io.gatling.core.Predef._
+import io.gatling.http.Predef._
+
+class Login extends Simulation{
+  val url = "http://performance.jtalks.org/jcommune"
```

Comment:
I think that all constants shold be placed in another class (e.g. Utils) and passed as an arguments to JVM. In this case configuring the tests will be more simple.